examples/configure-ingress-secrets-manager/README.md

```diff
@@ -0,0 +1,57 @@
+# IBM Cloud Kubernetes and IBM Cloud Secrets Manager integration sample
+
```

Comment:
It is a generic comment

Make sure both examples and modules directory have following set of files

1) main.tf
2) variables.tf
3) versions.tf
4) README.md
5) input.tfvars / output.tf

Reference

https://github.com/terraform-ibm-modules/terraform-ibm-cluster/tree/master/examples/classic-kubernetes-multi-zone

https://github.com/terraform-ibm-modules/terraform-ibm-cluster/tree/master/modules/classic-kubernetes-multi-zone